freqtrade/tests/exchange/test_exchange.py

```diff
@@ -14,20 +14,43 @@
 from freqtrade.tests.conftest import log_has, get_patched_exchange
 
 
+def ccxt_exceptionhandlers(mocker, default_conf, api_mock, fun, innerfun, **kwargs):
```

Comment:
I personally find `innerfun` a little bit misleading, maybe `mock_ccxt_fun` or something like that is more appropriate

Reply:
cought me on my weakest side - naming things